supervisor/web.py

```diff
@@ -429,9 +433,7 @@ def render(self):
                 if message is NOT_DONE_YET:
                     return NOT_DONE_YET
                 if message is not None:
-                    server_url = form['SERVER_URL']
-                    location = server_url + '?message=%s' % urllib.quote(
-                        message)
+                    location = '%s?message=%s' % (self.base_path, urllib.quote(message))
```

Comment:
[This is probably fine](http://greenbytes.de/tech/webdav/draft-ietf-httpbis-p2-semantics-17.html#rfc.section.9.5) but it should be noted that this PR quietly removed `SERVER_URL` from the redirect.